metabase/metabase
# Mantine Switch

Closes https://github.com/metabase/metabase/issues/33993

### Description

Implements the [Mantine Switch component](https://v6.mantine.dev/core/switch/) in our `metabase/ui` component library.

### How to verify

1. `yarn storybook`
2. http://localhost:6006/?path=/story/inputs-switch--default-story
3. Confirm it looks and behaves like the Figma

### Demo

<img width="1055" alt="Screenshot 2023-10-03 at 4 18 51 PM" src="https://github.com/metabase/metabase/assets/37751258/2e7be091-8e6e-47df-b771-7d856457d4ec">

### Checklist


_Not applicable to component library._
